examples/aws/storage-efs/README.md

```diff
@@ -0,0 +1,225 @@
+# Deploying Kubeflow with AWS EFS as Persistent Storage
+
+This guide describes how to deploy Kubeflow on AWS EKS using EFS as the Persistent Storage. <complete intro>
+
+## 1.0 Prerequisites
+
+1. Install the CLI tools
+2. Clone this repo and checkout the right branch
+3. Create an EKS cluster and
+4. Make sure the following environment variables are set - 
+```
+export CLUSTER_NAME=<clustername>
+export CLUSTER_REGION=<clusterregion>
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+```
+
+## 2.0 Install the EFS CSI Driver
+We recommend installing the EFS CSI Driver v1.3.4 directly from the [the aws-efs-csi-driver github repo](https://github.com/kubernetes-sigs/aws-efs-csi-driver) as follows - 
+
+```
+kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=tags/v1.3.4"
+```
+
+You should see the following resources get created - 
+```
+serviceaccount/efs-csi-controller-sa created
+serviceaccount/efs-csi-node-sa created
+clusterrole.rbac.authorization.k8s.io/efs-csi-external-provisioner-role created
+clusterrolebinding.rbac.authorization.k8s.io/efs-csi-provisioner-binding created
+deployment.apps/efs-csi-controller created
+daemonset.apps/efs-csi-node created
+csidriver.storage.k8s.io/efs.csi.aws.com configured
+```
+
+Additionally, you can confirm that EFS CSI Driver was installed into the default kube-system namespace for you. You can check using the following command - 
+```
+kubectl get csidriver
+
+NAME              ATTACHREQUIRED   PODINFOONMOUNT   MODES        AGE
+efs.csi.aws.com   false            false            Persistent   5d17h
+```
+
+## 3.0 Create the IAM Policy for the CSI Driver
+The driver requires IAM permission to talk to Amazon EFS to manage the volume on user's behalf. Here, we will be creating/annotating the Service Account `efs-csi-controller-sa` with an IAM Role which has the required permissions.
```

Comment:
Let's be precise here....

 CSI driver's service account (created during installation) requires IAM permission to make calls to AWS APIs on your behalf.